src/foundry/foundry.js/pixi/containers/placeableObjects/token.d.ts

```diff
@@ -159,6 +107,11 @@ declare class Token extends PlaceableObject<Token.Data> {
    */
   get inCombat(): boolean;
 
+  /**
+   * Return a reference to a Combatant that represents this Token, if one is present in the current encounter.
+   */
+  get combatant(): Combatant | null;
```

Comment:
`Combatant` does not exist currently. Where should this class be placed?

Reply:
Have a look here: https://github.com/League-of-Foundry-Developers/foundry-vtt-types/blob/foundry-0.8.x/src/foundry/index.d.ts#L136
For classes from `foundry.js` we group them by inheritance, so since `Combatant` is extending the `ClientDocumentMixin`, it should go with the others.